packages/sw-offline-google-analytics/test/browser/enqueue-request.js

```diff
@@ -17,9 +17,9 @@
 'use strict';
 
 describe('enqueue-request', () => {
-  const enqueueRequest = goog.offlineGoogleAnalytics.test.enqueueRequest;
-  const constants = goog.offlineGoogleAnalytics.test.constants;
-  const IDBHelper = goog.offlineGoogleAnalytics.test.IDBHelper;
+  const enqueueRequest = goog.offlineGoogleAnalytics.test.EnqueueRequest;
```

Comment:
These aren't part of the public interface. These are exports consumed via the tests, and the tests were also updated to account for the new naming.

(The motivation for renaming is automation—instead of having to manually create the exported names, they're [generated](https://github.com/GoogleChrome/sw-helpers/pull/485/files#diff-96ddb612cfff564769e651b776a01117R37) by running the individual source file through [`uppercamelcase`](https://www.npmjs.com/package/uppercamelcase).)